tests/contest/runtimetest/src/tests.rs

```diff
@@ -546,6 +548,34 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
     }
 }
 
+pub fn validate_process(spec: &Spec) {
+    let process = spec.process().as_ref().unwrap();
+
+    if process.cwd().ne(&getcwd().unwrap()) {
+        eprintln!(
+            "error due to spec cwd want {:?}, got {:?}",
+            process.cwd(),
+            getcwd().unwrap()
+        )
+    }
+
+    if env::var("testa").unwrap().to_string().ne("valuea") {
+        eprintln!(
+            "error due to spec environment value of testa want {:?}, got {:?}",
+            "valuea",
+            env::var("testa")
+        )
+    }
+
+    if env::var("testb").unwrap().to_string().ne("123") {
+        eprintln!(
+            "error due to spec environment value of testb want {:?}, got {:?}",
+            "123",
+            env::var("testb")
+        )
+    }
+}
```

Comment:
I think for this as well, we should cross check this with env from the spec, similar to how the go test is doing.

Reply:
I fixed code like this.

https://github.com/youki-dev/youki/pull/2968/commits/bd0bbb1d707c9b329d1eb7b689c72662eb51b8c5